examples/measures-trajectories-cats-kerr.ipynb

```diff
@@ -0,0 +1,731 @@
DRBWyqAb5nZiaSbef3/zOzEzK7M7OrVL36xprsDAACo3tZXAFNmdqTQFDylKviOMEr4VJL2nj3zau8OAABgfVoTAN39XNK5mXXM7FrSp+5+U/d9AQAArNvGBEAz6xXY/c7dcwd5uPvEzEaSvpb0pJSbAwAArWJme5L2JY0l9SSN3f1yzjFDSV+GfacORg1T2O2FH3ck3WbnLw7X70u6Dft8WaSwtREBMIS/YYFDXkqa1cfvUlLHzA7mvVkAAACxNJe4+2G07czMxu4+nnHonqTnYf/sY2N33w3BTvFYBTM7MrOTNASGgDjIXP/azI7nXP+tjQiA4cnM7beXFd6ga0nPprwgnVXvDQAAtE5fD2cTGSkpVs3KK2Ml4xHu9O50dAeSrtJzu/s7cxW7+7mZnSmMTQjXys5n/HnYNtAC2jAK+ErJCx1Lm5PpAwgAAIo60sMMcRW2z3Lt7pfufuPu4/RLkqLm2/0p3d46kmRmHYUm58zjNwrVxUVsdQAML+pFzkMDSS8WLZMCAABI7wSwd4pLaZ++WWMWsv34wv4nme2Xki7SpuCwz5Gks/Bjev5scesuur+5NqIJeBXu/iLM6bcr6bWkXUlneW8CAADYKk/N7Cr6+bSE//93pfvAlyOvOpcrhLz4/uTug9DH79rMBum5ovtOz93Vu83I3Snbc219AJTyEzcAAGiG7777rf7tv/urKk79yt33Sz5nmeMHDvIWpnD3Z2Z2oaRP4Y2kj6LHJmZ2qWRASRw0e9H3uQF0q5uAAQAAmig06+YGtbBqWV/JgJGekmpg3LTcVzQIJNPsm20aztWKCiAAAEDWKnMMm1ln1lx+C/hMUWUvOu9IyRQzY0ljM/tASf+/CyXd2OTuYzM7DiFSSpp80zC5PdPAAAAAlGmFOYZz++BFVbi5ASzsu5cNkOk54kGqYZ/DMM/f2/mLw/bz6Ni9aPtcBEAAANA6y84xHPrgjfWwL2BX0mTBGUYOlD9QY1/J/MV5RjnXzB57PuPxd9AHEAAAoJhLJYErthe2L+JD5VcKx5KeTTmmozD3oJldhH6CsYEWnARaIgACAAAUNdDD6uE7q3CYWcfMbnOCmpQzj6B03/QbpoF5KzRX70TVxYmisGlmzyWNisxvTBMwAABAAaEZeBCC11hJoBvmBLDuw6OlcExuc66798P8xUMl8xdLSdNyXN0bSDoys53w823RKe8IgAAAAAWFpdumLikbBmM8mfLYzKbaeWEuBM0H8wcWQRMwAABAyxAAAQAAWoYACAAA0DIEQAAAgJYhAAIAALQMARAAAKBlCIAAAAAtQwAEAABoGQIgAABAyxAAAQAAWoYACAAA0DIEQAAAgJYhAAIAALQMARAAAKBlHtV9AwAAoN3+5re/1f/2V9/VfRutQgUQAACgZQiAAAAALUMABAAAaBkCIAAAQMsQAAEAAFqGAAgAANAyBEAAAICWIQACAAC0DAEQAACgZQiAAAAALUMABAAAaBkCIAAAQMsQAAEAAFqGAAgAANAyBEAAAICWIQACAAC0zKO6b2DdzKwjaeju/brvBQAAbDYzO5I0cffLBfffk7QvaSypJ2mcPbasfWZpXQCUNJTUrfsmAADAZjOzA0lfSDpecP+ekiLUYbTtzMzG7j4uc595WtUEHF4wwh8AAFiamfXMbKSk8nZX4NC+pFFm20hJcarsfWZqVQCUdCDpou6bAAAAm8vdx+7ed/fTgoceSbrJbLsK28veZ6bWBMBQpv2q7vsAAADtE8YgPKgYuvskPN4ra59F7qdNfQA77j4xs4V2NrMTSSfhx799/N57/2tld4YqPZX0qu6bwNJ4/zYX791m++N1Xuzf+3f/8vPvxk8rOPXvmdlV9PPpElW7snSl+6CW421wK2Gfuf0AWxEAzezI3c+LHBN+QU7D8Vfuvl/JzaFSvHebjfdvc/HebbZMaKqcu//9dV6vJp017jPX1jcBh1LptJQMAADQOhtTAVy0TTu4i0qjH9dY7gUAAA21QrZY5Zqdeecpa59ZNiIApvPdFDjkpaQXYZLEMsrYBMjNxXu32Xj/Nhfv3Wbb+vdv2WyxwiXTfnldRS2ToaUyffyupH3mMncvcO+bJQzk2M1s3lPSQfJc0suifQMBAABSZnYrqb/IKhxh32N3v4m29SRdu/uTMveZZyMqgMvKa/o1s+eSPnT3QQ23BAAA2utSyfJt8Rx+e2F72fvMtPWDQHLs1H0DAABga3SVMzLXzDpmdhtaI1MDPVw2rh+2l73PTFvdBBwLpdGBpI+VvFGnkkZx+RRA/VZd4Bz1MrMjJe/bbvg+oqvNZgp9yobu3q/7XpomvDafKfkdP1LyeXUp6SL9fQ/7fCNpELdIhs+4A91/xt1kP+PK2mfmc2hLAATQfOEPtVF2gXMlH6ALdWxGfUL4G6d/WIf/AV4rCRFbP6hg24S1brvunq00YQu0sQm4NKG8m12MGQ1mZkdm9tzMRmZ2Ef6HheZYeYFz1KoXt6qEKSqGevieouHCH2Pduu8D1dnqQSBrMBT/gWyMqDoRl+evzaxLdaIxjvQwLFxJuqjhXlBA+O/pEzM7zcxNdhke71HF3SgHSv67O5y3IzYTFcAl8dfRRqI60WBlLXCOeoT3qadorVJsJjM7kPRV3feBahEAl5f+dYQNEFUnsiO13lYn1n9XyFh4oXQ0k7s/yRlYtydpQvVvo6y0wgQ2AwFwCfx1tHmoTmyEUhY4R+N8Junzum8CizGzI0ZttwN9AJfTcfeJmdV9HyhgyuzoVCeAioT5z+7cfZXls7AmoYWEyl9LUAEsiL+Otg7ViYbJaabHBgrdKvrxlD5ovI+Zc7M9CIAF8NfRdqE60TjxQulvFV3gHI0xlPRR3TeBxYRJha/qvg+sTyubgAt2+L+LOsN+zHQh9Vvh/cueo+/uz8q7M6widKsY62FfwK5opt8oYX7UTxlIsFH2Je2a2SfRtj1JPTMbSnpJ69d2aV0ADP/jLzKp7EtJL/jrq
```

Comment:
@fminga could you just change "Linblad"->"Lindblad"? Thanks

Reply:
Done! Thx for the critical reading!